src/app/wagmiConfig.ts

```diff
@@ -1,9 +1,10 @@
 import { http, createConfig } from "wagmi";
-import { mainnet, optimism } from "wagmi/chains";
+import { mainnet, optimism, optimismSepolia } from "wagmi/chains";
 
 export const WagmiConfig = createConfig({
-  chains: [optimism],
+  chains: [optimism, optimismSepolia],
```

Comment:
On prod we should only support `optimism`. Can we somehow make it dependant on some env variables?